src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/GenerateComponentLocationAnalysisOperation.java

```diff
@@ -76,4 +95,17 @@ private void runComponentLocator(List<Component> componentsList, File scanOutput
     private Input generateComponentLocatorInput(List<Component> componentsList, File sourceDir) {
         return new Input(sourceDir.getAbsolutePath(), new JsonObject(), componentsList);
     }
+
+
+    private void serializeInputToJson(File saveInputFileDir, Input libInput) {
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        String serializedLibInput = gson.toJson(libInput);
+        try {
+            File componentsSourceInputFile =  new File (saveInputFileDir, "components-source.json");
+            DetectFileUtils.writeToFile(componentsSourceInputFile, serializedLibInput);
+        } catch (IOException ex) {
+            logger.info("UH OH SPAGHETTIOO");
+//            throw new DetectUserFriendlyException("Failed to create component location analysis output file", ex, ExitCodeType.FAILURE_UNKNOWN_ERROR);
+        }
+    }
```

Comment:
Please make sure to pull from my PR branch on input file creation to avoid a merge conflict. My PR is specific about the input file creation even though the code is the same.

Reply:
Oops, did not mean to include this in the PR at all. Removed, thank you for pointing it out!